client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.jsx

```diff
@@ -0,0 +1,308 @@
+					{ value }
+				</Badge>
+			);
+			break;
+		}
+		case 'warning': {
+			content = (
+				<Badge className="sites-overview__badge" type="warning">
+					{ value }
+				</Badge>
+			);
+			break;
+		}
+		case 'success': {
+			content = <Gridicon icon="checkmark" size={ 18 } className="sites-overview__grey-icon" />;
+			break;
+		}
+		case 'active': {
+			content = <Gridicon icon="minus-small" size={ 18 } className="sites-overview__icon-active" />;
+			break;
+		}
+		case 'progress': {
+			content = (
+				<svg
+					className="sites-overview__vertical-align-middle"
+					width="16"
+					height="16"
+					viewBox="0 0 16 16"
+					fill="none"
+					xmlns="http://www.w3.org/2000/svg"
+				>
+					<circle cx="8" cy="8" r="7.25" stroke="black" strokeWidth="1.5" />
+					<rect x="7" y="4" width="1.5" height="5" fill="#1E1E1E" />
+					<rect
+						x="10.8901"
+						y="10.77"
+						width="1.5"
+						height="4"
+						transform="rotate(135 10.8901 10.77)"
+						fill="#1E1E1E"
+					/>
+				</svg>
+			);
+			break;
+		}
+		case 'inactive': {
+			content = (
+				<span className="sites-overview__status-add-new">
+					<Gridicon icon="plus-small" size={ 16 } />
+					<span>{ translate( 'Add' ) }</span>
+				</span>
+			);
+			break;
+		}
+	}
+	return (
+		<StatusContent
+			content={ content }
+			tooltip={ tooltip }
+			tooltipId={ tooltipId }
+			link={ link }
+			siteError={ siteError }
+		/>
+	);
+};
+
+const siteFormatter = ( rows ) => {
+	const { row } = getRowMetaData( rows, 'site' );
+	const site = row.value;
+	const value = site.url;
+
+	return <span className="sites-overview__row-text">{ value }</span>;
+};
+
+/**
+ * Returns formatted sites
+ *
+ * @param  {data} data Object that holds sites as items
+ * @returns {Array} An empty array if no sites are found or
+ * formatted sites based on status and feature type
+ */
+
+export const formatSites = ( data ) => {
+	const sites = data?.items;
+	if ( ! Array.isArray( sites ) ) {
+		return [];
+	}
+	return sites.map( ( site ) => {
+		const pluginUpdates = site.awaiting_plugin_updates;
+		let scanValue = '';
+		if ( site.latest_scan_status === 'failed' ) {
+			scanValue = translate( 'Failed' );
+		}
+		const scanThreats = site.latest_scan_threats_found.length;
+		if ( scanThreats > 0 ) {
+			scanValue =
+				scanThreats > 1
+					? translate( '%(threats)d Threats', {
+							args: {
+								threats: scanThreats,
+							},
+					  } )
+					: translate( '%(threats)d Threat', {
+							args: {
+								threats: scanThreats,
+							},
+					  } );
+		}
+		let error = '';
+		if (
+			! site.is_connection_healthy ||
+			! site.access_xmlrpc ||
+			! site.valid_xmlrpc ||
+			! site.authenticated_xmlrpc ||
+			! site.has_credentials
+		) {
+			error = translate( 'FIX CONNECTION' );
+		}
+		return {
+			site: {
+				value: site,
+				formatter: siteFormatter,
+				error,
+			},
+			backup: {
+				value: site.latest_backup_status === 'failed' ? translate( 'Failed' ) : '',
```

Comment:
This is just a nitpick, but usually we would do `'failed' === site.latest_backup_status`, putting the constant first to avoid accidental assignment of a variable. I believe in PHP this is checked by the linter and even though it isn't checked in JS we still follow that convention. Same for some of the comparisons below.